.et

```diff
@@ -0,0 +1,3 @@
+{ "bad_versions" : [ "1.2.1",
+                     "1.2.3"]
```

Comment:
this should be checked and updated before release.

Reply:
I think these are legitimately buggy. We can really only plausibly check back to 1.3.0, since this functionality will be missing before that. Is it worth looking for critical bugs in earlier [release logs](https://github.com/nipy/nipype/releases)?

Reply:
this was more about whether even these two were correct. from a usage perspective, nothing will change till the server starts reading these `.et` files and returning the info. i'll update the server soon.